typescript-light-bulb/main.ts

```diff
@@ -0,0 +1,15 @@
+const $lightBulb = document.querySelector('.bulb');
+const $background = document.querySelector('.back');
+
+if (!$lightBulb) throw new Error('$lightBulb query failed');
+if (!$background) throw new Error('$background query failed');
+
+$lightBulb.addEventListener('click', () => {
+  if ($lightBulb.className === 'bulb bulb-light') {
+    $lightBulb.className = 'bulb bulb-dark';
+    $background.className = 'back back-dark';
+  } else {
+    $lightBulb.className = 'bulb bulb-light';
+    $background.className = 'back back-light';
+  }
+});
```

Comment:
Rather than using the `className` to determine whether it is on or off, we should store a separate variable. The problem with using the `className` like this would be if there was a very long `className` you are checking.

Instead let's have a variable that is a _data type_ that is good for conditions with only two options.